src/pages/SearchMobilePage/SearchMobilePage.js

```diff
@@ -1,25 +1,112 @@
-import React, { useState } from 'react'
+import React, { useEffect, useState } from 'react'
 import cx from 'classnames'
 import { Link } from 'react-router-dom'
+import Svg from 'webkit/ui/Svg/react'
+import { Icon } from '@santiment-network/ui'
 import Tabs from '@santiment-network/ui/Tabs'
-import Icon from '@santiment-network/ui/Icon'
+import { client } from '../../apollo'
 import MobileHeader from './../../components/MobileHeader/MobileHeader'
-import Search from '../../components/Search/SearchContainer'
+import SearchBar from './SearchBar'
 import { TABS } from '../../components/Search/tabs'
-import {
-  getRecentAssets,
-  getRecentTrends,
-  removeRecentTrends,
-  removeRecentAssets,
-} from '../../utils/recent'
-import { safeDecode } from '../../utils/utils'
+import { useTabOptions } from './utils'
 import styles from './SearchMobilePage.module.scss'
+import PageLoader from '../../components/Loader/PageLoader'
+
+const AlternativeLink = ({ link, onClick, children }) => {
```

Comment:
Name `Link` from router as a `RouterLink`
```suggestion
const Link = ({ link, onClick, children }) => {
```